docs/README.md

```diff
@@ -99,11 +99,21 @@ apiVersion: manifests/v1beta1
 metadata:
   name: my-first-manifest
 spec:
+  sources:
+    charts:
+    - type: directory
+      name: local
+      location: ./charts
+    - type: repo
+      name: hashicorp
+      location: https://helm.releases.hashicorp.com
   charts:
   - name: consul
-    version: 9.2.8
+    source: hashicorp
+    version: 0.33.0
```

Comment:
It looks like we're moving away from the bitnami consul and switching to the official hashicorp consul?  Am I reading this right?

Reply:
This is just documentation with examples really. I had the hashicorp-based consul in a newer manifest format handy which is really the only reason we see this change.